examples/slack-connect-example/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Nit: we should use the shared eslint config here like the existing nextjs-example